lib/scrabble.rb

```diff
@@ -0,0 +1,52 @@
+module Scrabble
+  class Scrabble
```

Comment:
This is a bit confusing where the name of the module is the same as the class, so I'd recommend a minor re-name if possible